test/core/store/fs/test_registry.py

```diff
@@ -341,7 +342,7 @@ def _assert_dataset_supported(
             requested_dtype_alias: Optional[str],
             expected_dtype_aliases: Set[str],
             expected_return_type: Union[Type[xr.Dataset],
-                                        Type[MultiLevelDataset]],
+            Type[MultiLevelDataset]],
```

Comment:
I believe the prior existing indent was correct.

Reply:
Yes, true and I didnt notice this change. I do it :)